src/setup_payload/tests/TestHelpers.h

```diff
@@ -34,8 +34,8 @@ const std::string kOptionalDefaultStringValue = "myData";
 const uint8_t kOptionalDefaultIntTag    = 3;
 const uint32_t kOptionalDefaultIntValue = 12;
 
-const char * kSerialNumberDefaultStringValue   = "123456789";
-const uint32_t kSerialNumberDefaultUInt32Value = 123456789;
+const std::string kSerialNumberDefaultStringValue = "123456789";
```

Comment:
Please don't change this to string... the string constructor isn't constexpr.

Reply:
But do we need constexpr here? This variable is used only as function arguments the same as kOptionalDefaultStringValue which is const string

Reply:
If it's not constexpr there is a hidden runtime cost to constructing it. This module probably shouldn't use strings in the first palce, but definitely don't use strings for global variables. You should do this instead:

`constexpr char kSerialNumberDefaultStringValue[] = "123456789";`

It avoids the runtime costs.

Reply:
Right, now I understand the case